after major rewrite

djdanielsson opened the pull request:
all the other little stuff

djdanielsson committed:
deleted:    roles/controller_dispatch/README.md
	deleted:    roles/controller_dispatch/defaults/main.yml
	deleted:    roles/controller_dispatch/meta/argument_specs.yml
	deleted:    roles/controller_dispatch/meta/main.yml
	deleted:    roles/controller_dispatch/tasks/main.yml
	deleted:    roles/controller_dispatch/tests/configs/inventories.yml
	deleted:    roles/controller_dispatch/tests/configs/projects.yml
	deleted:    roles/controller_dispatch/tests/test.yml
	deleted:    roles/controller_filetree_create/README.md
	deleted:    roles/controller_filetree_create/automatetheautomation.md
	deleted:    roles/controller_filetree_create/defaults/main.yml
	deleted:    roles/controller_filetree_create/handlers/main.yml
	deleted:    roles/controller_filetree_create/meta/argument_specs.yml
	deleted:    roles/controller_filetree_create/meta/main.yml
	deleted:    roles/controller_filetree_create/tasks/all.yml
	deleted:    roles/controller_filetree_create/tasks/applications.yml
	deleted:    roles/controller_filetree_create/tasks/constructed_inventory.yml
	deleted:    roles/controller_filetree_create/tasks/credential_types.yml
	deleted:    roles/controller_filetree_create/tasks/credentials.yml
	deleted:    roles/controller_filetree_create/tasks/execution_environments.yml
	deleted:    roles/controller_filetree_create/tasks/groups.yml
	deleted:    roles/controller_filetree_create/tasks/hosts.yml
	deleted:    roles/controller_filetree_create/tasks/instance_groups.yml
	deleted:    roles/controller_filetree_create/tasks/inventory.yml
	deleted:    roles/controller_filetree_create/tasks/inventory_sources.yml
	deleted:    roles/controller_filetree_create/tasks/job_templates.yml
	deleted:    roles/controller_filetree_create/tasks/labels.yml
	deleted:    roles/controller_filetree_create/tasks/main.yml
	deleted:    roles/controller_filetree_create/tasks/notification_templates.yml
	deleted:    roles/controller_filetree_create/tasks/organizations.yml
	deleted:    roles/controller_filetree_create/tasks/projects.yml
	deleted:    roles/controller_filetree_create/tasks/schedules.yml
	deleted:    roles/controller_filetree_create/tasks/settings.yml
	deleted:    roles/controller_filetree_create/tasks/team_roles.yml
	deleted:    roles/controller_filetree_create/tasks/teams.yml
	deleted:    roles/controller_filetree_create/tasks/user_roles.yml
	deleted:    roles/controller_filetree_create/tasks/users.yml
	deleted:    roles/controller_filetree_create/tasks/workflow_job_templates.yml
	deleted:    roles/controller_filetree_create/templates/current_applications.j2
	deleted:    roles/controller_filetree_create/templates/current_credential_types.j2
	deleted:    roles/controller_filetree_create/templates/current_credentials.j2
	deleted:    roles/controller_filetree_create/templates/current_execution_environments.j2
	deleted:    roles/controller_filetree_create/templates/current_groups.j2
	deleted:    roles/controller_filetree_create/templates/current_hosts.j2
	deleted:    roles/controller_filetree_create/templates/current_instance_groups.j2
	deleted:    roles/controller_filetree_create/templates/current_inventories.j2
	deleted:    roles/controller_filetree_create/templates/current_inventory_sources.j2
	deleted:    roles/controller_filetree_create/templates/current_job_templates.j2
	deleted:    roles/controller_filetree_create/templates/current_labels.j2
	deleted:    roles/controller_filetree_create/templates/current_notification_templates.j2
	deleted:    roles/controller_filetree_create/templates/current_organizations.j2
	deleted:    roles/controller_filetree_create/templates/current_projects.j2
	deleted:    roles/controller_filetree_create/templates/current_schedules.j2
	deleted:    roles/controller_filetree_create/templates/current_settings.j2
	deleted:    roles/controller_filetree_create/templates/current_team_roles.j2
	deleted:    roles/controller_filetree_create/templates/current_teams.j2
	deleted:    roles/controller_filetree_create/templates/current_user_roles.j2
	deleted:    roles/controller_filetree_create/templates/current_users.j2
	deleted:    roles/controller_filetree_create/templates/current_workflow_job_templates.j2
	deleted:    roles/controller_filetree_create/tests/.gitignore
	deleted:    roles/controller_filetree_create/tests/filetree_create.yml
	deleted:    roles/controller_filetree_create/tests/inventory
	deleted:    roles/controller_filetree_create/vars/main.yml
	deleted:    roles/controller_filetree_read/README.md
	deleted:    roles/controller_filetree_read/defaults/main.yml
	deleted:    roles/controller_filetree_read/handlers/main.yml
	deleted:    roles/controller_filetree_read/meta/argument_specs.yml
	deleted:    roles/controller_filetree_read/meta/main.yml
	deleted:    roles/controller_filetree_read/tasks/applications.yml
	deleted:    roles/controller_filetree_read/tasks/credential_input_sources.yml
	deleted:    roles/controller_filetree_read/tasks/credential_types.yml
	deleted:    roles/controller_filetree_read/tasks/credentials.yml
	deleted:    roles/controller_filetree_read/tasks/execution_environments.yml
	deleted:    roles/controller_filetree_read/tasks/groups.yml
	deleted:    roles/controller_filetree_read/tasks/hosts.yml
	deleted:    roles/controller_filetree_read/tasks/instance_groups.yml
	deleted:    roles/controller_filetree_read/tasks/inventories.yml
	deleted:    roles/controller_filetree_read/tasks/inventory_sources.yml
	deleted:    roles/controller_filetree_read/tasks/job_templates.yml
	deleted:    roles/controller_filetree_read/tasks/labels.yml
	deleted:    roles/controller_filetree_read/tasks/main.yml
	deleted:    roles/controller_filetree_read/tasks/notifications.yml
	deleted:    roles/controller_filetree_read/tasks/organizations.yml
	deleted:    roles/controller_filetree_read/tasks/projects.yml
	deleted:    roles/controller_filetree_read/tasks/roles.yml
	deleted:    roles/controller_filetree_read/tasks/schedules.yml
	deleted:    roles/controller_filetree_read/tasks/settings.yml
	deleted:    roles/controller_filetree_read/tasks/teams.yml
	deleted:    roles/controller_filetree_read/tasks/user_accounts.yml
	deleted:    roles/controller_filetree_read/tasks/workflow_job_templates.yml
	deleted:    roles/controller_filetree_read/tests/.gitignore
	deleted:    roles/controller_filetree_read/tests/config-controller-filetree.yml
	deleted:    roles/controller_filetree_read/tests/inventory
	deleted:    roles/controller_filetree_read/vars/main.yml
	deleted:    roles/controller_object_diff/README.md
	deleted:    roles/controller_object_diff/defaults/main.yml
	deleted:    roles/controller_object_diff/handlers/main.yml
	deleted:    roles/controller_object_diff/meta/argument_specs.yml
	deleted:    roles/controller_object_diff/meta/main.yml
	deleted:    roles/controller_object_diff/tasks/applications.yml
	deleted:    roles/controller_object_diff/tasks/credential_types.yml
	deleted:    roles/controller_object_diff/tasks/credentials.yml
	deleted:    roles/controller_object_diff/tasks/execution_environments.yml
	deleted:    roles/controller_object_diff/tasks/groups.yml
	deleted:    roles/controller_object_diff/tasks/hosts.yml
	deleted:    roles/controller_object_diff/tasks/instance_groups.yml
	deleted:    roles/controller_object_diff/tasks/inventories.yml
	deleted:    roles/controller_object_diff/tasks/inventory_sources.yml
	deleted:    roles/controller_object_diff/tasks/job_templates.yml
	deleted:    roles/controller_object_diff/tasks/main.yml
	deleted:    roles/controller_object_diff/tasks/notification_templates.yml
	deleted:    roles/controller_object_diff/tasks/organizations.yml
	deleted:    roles/controller_object_diff/tasks/projects.yml
	deleted:    roles/controller_object_diff/tasks/roles.yml
	deleted:    roles/controller_object_diff/tasks/schedules.yml
	deleted:    roles/controller_object_diff/tasks/teams.yml
	deleted:    roles/controller_object_diff/tasks/user_accounts.yml
	deleted:    roles/controller_object_diff/tasks/workflow_job_templates.yml
	deleted:    roles/controller_object_diff/tests/.gitignore
	deleted:    roles/controller_object_diff/tests/drop_diff.yml
	deleted:    roles/controller_object_diff/tests/inventory
	deleted:    roles/controller_object_diff/tests/object_diff.yml
	deleted:    roles/controller_object_diff/vars/main.yml
	renamed:    roles/controller_token/README.md -> roles/eda_controller_tokens/README.md
	renamed:    roles/controller_token/defaults/main.yml -> roles/eda_controller_tokens/defaults/main.yml
	renamed:    roles/controller_token/meta/argument_specs.yml -> roles/eda_controller_tokens/meta/argument_specs.yml
	renamed:    roles/controller_token/meta/main.yml -> roles/eda_controller_tokens/meta/main.yml
	renamed:    roles/controller_token/tasks/main.yml -> roles/eda_controller_tokens/tasks/main.yml
	renamed:    roles/controller_token/tests/test.yml -> roles/eda_controller_tokens/tests/test.yml
	renamed:    roles/controller_token/tests/vars/user_tokens.yml -> roles/eda_controller_tokens/tests/vars/user_tokens.yml
	renamed:    roles/eda_credential/README.md -> roles/eda_credentials/README.md
	renamed:    roles/eda_credential/defaults/main.yml -> roles/eda_credentials/defaults/main.yml
	renamed:    roles/eda_credential/meta/argument_specs.yml -> roles/eda_credentials/meta/argument_specs.yml
	renamed:    roles/eda_credential/meta/main.yml -> roles/eda_credentials/meta/main.yml
	renamed:    roles/eda_credential/tasks/main.yml -> roles/eda_credentials/tasks/main.yml
	renamed:    roles/eda_credential/tests/test.yml -> roles/eda_credentials/tests/test.yml
	renamed:    roles/eda_credential/tests/vars/credentials.yml -> roles/eda_credentials/tests/vars/credentials.yml
	renamed:    roles/eda_decision_environment/README.md -> roles/eda_decision_environments/README.md
	renamed:    roles/eda_decision_environment/defaults/main.yml -> roles/eda_decision_environments/defaults/main.yml
	renamed:    roles/eda_decision_environment/meta/argument_specs.yml -> roles/eda_decision_environments/meta/argument_specs.yml
	renamed:    roles/eda_decision_environment/meta/main.yml -> roles/eda_decision_environments/meta/main.yml
	renamed:    roles/eda_decision_environment/tasks/main.yml -> roles/eda_decision_environments/tasks/main.yml
	renamed:    roles/eda_decision_environment/tests/test.yml -> roles/eda_decision_environments/tests/test.yml
	renamed:    roles/eda_decision_environment/tests/vars/decision_environments.yml -> roles/eda_decision_environments/tests/vars/decision_environments.yml
	deleted:    roles/eda_dispatch/README.md
	deleted:    roles/eda_dispatch/defaults/main.yml
	deleted:    roles/eda_dispatch/meta/argument_specs.yml
	deleted:    roles/eda_dispatch/meta/main.yml
	deleted:    roles/eda_dispatch/tasks/main.yml
	deleted:    roles/eda_dispatch/tests/configs/credentials.yml
	deleted:    roles/eda_dispatch/tests/configs/projects.yml
	deleted:    roles/eda_dispatch/tests/test.yml
	renamed:    roles/eda_project/README.md -> roles/eda_projects/README.md
	renamed:    roles/eda_project/defaults/main.yml -> roles/eda_projects/defaults/main.yml
	renamed:    roles/eda_project/meta/argument_specs.yml -> roles/eda_projects/meta/argument_specs.yml
	renamed:    roles/eda_project/meta/main.yml -> roles/eda_projects/meta/main.yml
	renamed:    roles/eda_project/tasks/main.yml -> roles/eda_projects/tasks/main.yml
	renamed:    roles/eda_project/tests/test.yml -> roles/eda_projects/tests/test.yml
	renamed:    roles/eda_project/tests/vars/projects.yml -> roles/eda_projects/tests/vars/projects.yml
	renamed:    roles/eda_rulebook_activation/README.md -> roles/eda_rulebook_activations/README.md
	renamed:    roles/eda_rulebook_activation/defaults/main.yml -> roles/eda_rulebook_activations/defaults/main.yml
	renamed:    roles/eda_rulebook_activation/meta/argument_specs.yml -> roles/eda_rulebook_activations/meta/argument_specs.yml
	renamed:    roles/eda_rulebook_activation/meta/main.yml -> roles/eda_rulebook_activations/meta/main.yml
	renamed:    roles/eda_rulebook_activation/tasks/main.yml -> roles/eda_rulebook_activations/tasks/main.yml
	renamed:    roles/eda_rulebook_activation/tests/test.yml -> roles/eda_rulebook_activations/tests/test.yml
	renamed:    roles/eda_rulebook_activation/tests/vars/rulebook_activations.yml -> roles/eda_rulebook_activations/tests/vars/rulebook_activations.yml
	renamed:    roles/eda_user/README.md -> roles/eda_users/README.md
	renamed:    roles/eda_user/defaults/main.yml -> roles/eda_users/defaults/main.yml
	renamed:    roles/eda_user/meta/argument_specs.yml -> roles/eda_users/meta/argument_specs.yml
	renamed:    roles/eda_user/meta/main.yml -> roles/eda_users/meta/main.yml
	renamed:    roles/eda_user/tasks/main.yml -> roles/eda_users/tasks/main.yml
	renamed:    roles/eda_user/tests/test.yml -> roles/eda_users/tests/test.yml
	renamed:    roles/eda_user/tests/vars/users.yml -> roles/eda_users/tests/vars/users.yml
	deleted:    roles/gateway_dispatch/README.md
	deleted:    roles/gateway_dispatch/defaults/main.yml
	deleted:    roles/gateway_dispatch/meta/argument_specs.yml
	deleted:    roles/gateway_dispatch/meta/main.yml
	deleted:    roles/gateway_dispatch/tasks/main.yml
	deleted:    roles/hub_dispatch/README.md
	deleted:    roles/hub_dispatch/defaults/main.yml
	deleted:    roles/hub_dispatch/meta/argument_specs.yml
	deleted:    roles/hub_dispatch/meta/main.yml
	deleted:    roles/hub_dispatch/tasks/main.yml
	deleted:    roles/hub_dispatch/tests/configs/images.yml
	deleted:    roles/hub_dispatch/tests/configs/registries.yml
	deleted:    roles/hub_dispatch/tests/configs/repositories.yml
	deleted:    roles/hub_dispatch/tests/test.yml
	deleted:    roles/hub_offline_sync/README.md
	deleted:    roles/hub_offline_sync/defaults/main.yml
	deleted:    roles/hub_offline_sync/meta/argument_specs.yml
	deleted:    roles/hub_offline_sync/meta/main.yml
	deleted:    roles/hub_offline_sync/tasks/main.yml
	deleted:    roles/hub_offline_sync/templates/ansible.cfg.j2
	deleted:    roles/hub_offline_sync/templates/collections.yml.j2
	deleted:    roles/hub_offline_sync/templates/collections_list.yml.j2
	deleted:    roles/hub_offline_sync/tests/test.yml
	deleted:    roles/hub_repository/README.md
	deleted:    roles/hub_repository/defaults/main.yml
	deleted:    roles/hub_repository/meta/argument_specs.yml
	deleted:    roles/hub_repository/meta/main.yml
	deleted:    roles/hub_repository/tasks/main.yml
	deleted:    roles/hub_repository/tests/test.yml
	deleted:    roles/hub_repository/tests/vars/repositories.yml
	deleted:    roles/hub_repository_sync/README.md
	deleted:    roles/hub_repository_sync/defaults/main.yml
	deleted:    roles/hub_repository_sync/meta/argument_specs.yml
	deleted:    roles/hub_repository_sync/meta/main.yml
	deleted:    roles/hub_repository_sync/tasks/main.yml
	deleted:    roles/hub_repository_sync/tests/test.yml
	deleted:    roles/hub_repository_sync/tests/vars/repositories.yml

djdanielsson committed:
renamed:    controller_groups/README.md -> controller_host_groups/README.md
	renamed:    controller_groups/defaults/main.yml -> controller_host_groups/defaults/main.yml
	renamed:    controller_groups/meta/argument_specs.yml -> controller_host_groups/meta/argument_specs.yml
	renamed:    controller_groups/meta/main.yml -> controller_host_groups/meta/main.yml
	renamed:    controller_groups/tasks/main.yml -> controller_host_groups/tasks/main.yml
	renamed:    controller_groups/tests/configs/group.yml -> controller_host_groups/tests/configs/group.yml
	renamed:    controller_groups/tests/test.yml -> controller_host_groups/tests/test.yml
	new file:   dispatch/README.md
	new file:   dispatch/defaults/main.yml
	new file:   dispatch/meta/argument_specs.yml
	new file:   dispatch/meta/main.yml
	new file:   dispatch/tasks/main.yml
	new file:   dispatch/tests/configs/credentials.yml
	new file:   dispatch/tests/configs/images.yml
	new file:   dispatch/tests/configs/inventories.yml
	new file:   dispatch/tests/configs/projects.yml
	new file:   dispatch/tests/configs/registries.yml
	new file:   dispatch/tests/configs/repositories.yml
	new file:   dispatch/tests/test.yml
	modified:   gateway_applications/README.md
	modified:   gateway_applications/meta/argument_specs.yml
	modified:   gateway_applications/tasks/main.yml
	modified:   gateway_applications/tests/configs/applications.yml
	modified:   gateway_applications/tests/test.yml
	modified:   gateway_authenticator_maps/meta/argument_specs.yml
	modified:   gateway_authenticator_maps/tasks/main.yml
	modified:   gateway_authenticators/meta/argument_specs.yml
	modified:   gateway_authenticators/tasks/main.yml
	modified:   gateway_http_ports/meta/argument_specs.yml
	modified:   gateway_http_ports/tasks/main.yml
	modified:   gateway_organizations/meta/argument_specs.yml
	modified:   gateway_organizations/tasks/main.yml
	modified:   gateway_role_user_assignments/meta/argument_specs.yml
	modified:   gateway_role_user_assignments/tasks/main.yml
	modified:   gateway_routes/meta/argument_specs.yml
	modified:   gateway_routes/tasks/main.yml
	modified:   gateway_service_clusters/meta/argument_specs.yml
	modified:   gateway_service_clusters/tasks/main.yml
	modified:   gateway_service_keys/meta/argument_specs.yml
	modified:   gateway_service_keys/tasks/main.yml
	modified:   gateway_service_nodes/meta/argument_specs.yml
	modified:   gateway_service_nodes/tasks/main.yml
	modified:   gateway_services/meta/argument_specs.yml
	modified:   gateway_services/tasks/main.yml
	renamed:    gateway_platform_settings/README.md -> gateway_settings/README.md
	renamed:    gateway_platform_settings/defaults/main.yml -> gateway_settings/defaults/main.yml
	renamed:    gateway_platform_settings/meta/argument_specs.yml -> gateway_settings/meta/argument_specs.yml
	renamed:    gateway_platform_settings/meta/main.yml -> gateway_settings/meta/main.yml
	renamed:    gateway_platform_settings/tasks/main.yml -> gateway_settings/tasks/main.yml
	modified:   gateway_teams/meta/argument_specs.yml
	modified:   gateway_teams/tasks/main.yml
	modified:   gateway_users/meta/argument_specs.yml
	modified:   gateway_users/tasks/main.yml
	new file:   global_vars/README.md
	new file:   global_vars/defaults/main.yml
	new file:   global_vars/meta/main.yml
	new file:   global_vars/vars/main.yml
	modified:   hub_collection/tasks/main.yml
	modified:   hub_collection_remote/tasks/main.yml
	modified:   hub_collection_repository/tasks/main.yml
	modified:   hub_collection_repository_sync/tasks/main.yml
	modified:   hub_ee_image/tasks/main.yml
	modified:   hub_ee_namespace/tasks/main.yml
	modified:   hub_ee_registry/tasks/main.yml
	modified:   hub_ee_registry_index/tasks/main.yml
	modified:   hub_ee_registry_sync/tasks/main.yml
	modified:   hub_ee_repository/tasks/main.yml
	modified:   hub_ee_repository_sync/tasks/main.yml
	modified:   hub_namespace/tasks/main.yml
	modified:   hub_publish/tasks/main.yml
	new file:   meta_dependency_check/README.md
	new file:   meta_dependency_check/defaults/main.yml
	new file:   meta_dependency_check/meta/main.yml
	new file:   meta_dependency_check/tasks/main.yml
	new file:   meta_dependency_check/tests/inventory
	new file:   meta_dependency_check/tests/test.yml

djdanielsson committed:
modified:   dispatch/defaults/main.yml

djdanielsson committed:
modified:   controller_ad_hoc_command/README.md
	modified:   controller_ad_hoc_command_cancel/README.md
	modified:   controller_applications/README.md
	modified:   controller_bulk_host_create/README.md
	modified:   controller_bulk_job_launch/README.md
	modified:   controller_credential_input_sources/README.md
	modified:   controller_credential_types/README.md
	modified:   controller_credentials/README.md
	modified:   controller_execution_environments/README.md
	modified:   controller_global_vars/README.md
	modified:   controller_host_groups/README.md
	modified:   controller_hosts/README.md
	modified:   controller_instance_groups/README.md
	modified:   controller_instances/README.md
	modified:   controller_inventories/README.md
	modified:   controller_inventory_source_update/README.md
	modified:   controller_inventory_sources/README.md
	modified:   controller_job_launch/README.md
	modified:   controller_job_templates/README.md
	modified:   controller_jobs_cancel/README.md
	modified:   controller_labels/README.md
	modified:   controller_license/README.md
	modified:   controller_meta_dependency_check/README.md
	modified:   controller_notification_templates/README.md
	modified:   controller_organizations/README.md
	modified:   controller_project_update/README.md
	modified:   controller_projects/README.md
	modified:   controller_roles/README.md
	modified:   controller_schedules/README.md
	modified:   controller_settings/README.md
	modified:   controller_teams/README.md
	modified:   controller_users/README.md
	modified:   controller_workflow_job_templates/README.md
	modified:   controller_workflow_launch/README.md
	modified:   gateway_applications/README.md
	modified:   global_vars/README.md
	modified:   meta_dependency_check/README.md
	modified:   ../.github/CONTRIBUTING.md
	modified:   ../.github/workflows/ci_standalone_versioned.yml
	modified:   ../.github/workflows/pre-commit.yml
	modified:   ../.github/workflows/release.yml
	modified:   ../.github/workflows/release_manual.yml
	modified:   ../CHANGELOG.rst
	modified:   ../EXPORT_README.md
	modified:   ../README.md
	modified:   ../changelogs/changelog.yaml
	modified:   ../docs/aap_config_as_code_public_meeting.ics
	modified:   ../galaxy.yml
	deleted:    ../plugins/lookup/controller_object_diff.py
	deleted:    ../plugins/modules/controller_export_diff.py
	modified:   ../tests/playbooks/README.md
	modified:   ../tests/templated_role_example/README.md
	modified:   ../tests/templates/vars/controller_credential_types.yml
	modified:   ../tests/templates/vars/controller_credentials.yml
	modified:   ../tests/templates/vars/controller_execution_environments.yml
	modified:   ../tests/templates/vars/controller_groups.yml
	modified:   ../tests/templates/vars/controller_hosts.yml
	modified:   ../tests/templates/vars/controller_instance_groups.yml
	modified:   ../tests/templates/vars/controller_inventories.yml
	modified:   ../tests/templates/vars/controller_inventory_sources.yml
	modified:   ../tests/templates/vars/controller_job_templates.yml
	modified:   ../tests/templates/vars/controller_labels.yml
	modified:   ../tests/templates/vars/controller_ldap_configuration.json
	modified:   ../tests/templates/vars/controller_notification_templates.yml
	modified:   ../tests/templates/vars/controller_organizations.yml
	modified:   ../tests/templates/vars/controller_organizations_with_hub.yml
	modified:   ../tests/templates/vars/controller_projects.yml
	modified:   ../tests/templates/vars/controller_roles.yml
	modified:   ../tests/templates/vars/controller_schedules.yml
	modified:   ../tests/templates/vars/controller_teams.yml
	modified:   ../tests/templates/vars/controller_users.yml
	modified:   ../tests/templates/vars/controller_workflows.yml

djdanielsson committed:
modified:   README.md
	deleted:    history.sh

sean-m-sullivan commented on roles/gateway_applications/README.md:
````diff
@@ -39,13 +128,26 @@ Options for the `applications_list` variable:
 
 ```json
  {
+<<<<<<< HEAD
````
Merge Conflict heads here

sean-m-sullivan commented on roles/dispatch/tests/test.yml:
```diff
@@ -0,0 +1,55 @@
+---
+<<<<<<< HEAD
```
More head fails

sean-m-sullivan commented:
Found 16 <<<<<<< HEAD in various files.

djdanielsson committed:
modified:   roles/dispatch/tests/configs/projects.yml
	modified:   roles/dispatch/tests/test.yml
	modified:   roles/gateway_applications/README.md
	modified:   roles/gateway_applications/tests/configs/applications.yml
	modified:   roles/gateway_applications/tests/test.yml

djdanielsson commented:
good catch, I missed those test files. should be clear now, we will probably need to rewrite a lot of the tests if not all of them.

djdanielsson committed:
modified:   README.md

felixkrohn commented:
I'm a bit surprised the roles `controller_dispatch`, `controller_filetree_create` and `controller_object_diff` have been deleted with this PR. Is it planned to replace them by something else, should that code be maintained outside of this tree in future or are there any other reasonings I fail to understand?

Tompage1994 commented:
Hi, 

Note that this collection is going through a redesign. All the AAP components are being bought into this collection and some roles are being removed and placed into here: https://github.com/redhat-cop/aap_configuration_extended

Further communications will follow on this issue once the work has been done to change everything
